meta-resin-common/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -24,5 +24,6 @@ do_install_append() {
 
     install -d ${D}${sysconfdir}/default
     echo 'DROPBEAR_PORT="22222"' >> ${D}/etc/default/dropbear # Change default dropbear port to 22222
+    echo 'DROPBEAR_RSAKEY_DIR=/var/lib/dropbear' >> ${D}/etc/default/dropber # Change default dropbear host key location
```

Comment:
When the discussion was live, this was the decision that we made. To put the host key in var/lib/dropbear.